### DISCUSSION TITLE
Timezone not update

### DISCUSSION THREAD

**Discussion by cloud9-kj** [google/timesketch#1818] [Q&A, quick solutions, support]
Hi, 

The time is Timesketch is 1 hour late to the time of the server. 

How can I update the time of the server?

Thanks

**Comment by kiddinn** [comment]
can you clarify what you mean by `the time is Timesketch is 1 hour late to the time of the server`?

Is the clock on the container off by an hour compared to the host machine? or?

**Reply by cloud9-kj** [reply]
e.g. server's local time is 13:00:00, and when looking at the time on Timesketch UI, the time is recorded 1 hour early, 12:00:00.
